.github/checkgroup.yml

```diff
@@ -263,13 +264,21 @@ subprojects:
       - "app-examples (windows-2022, app, 3.9, oldest)"
       - "app-examples (windows-2022, lightning, 3.9, latest)"
 
+  - id: "lightning: Flagships"
+    paths:
+      - ".github/workflows/ci-flagship-apps.yml"
```

Comment:
Add `.github/actions/prep-apps/action.yml`?

Reply:
I would keep it light as the only command we use here is already tested on several other places... as required :chipmunk: